ascmhl/commands.py

```diff
@@ -316,7 +324,9 @@ def verify(
     """
 
     if packing_list is not None:
-        verify_entire_folder(root_path, verbose, packing_list, ignore_list, ignore_spec_file, calculate_only)
+        verify_entire_folder(
```

Comment:
would be good to make use of the named parameters rather than positional arguments here since there are so many params.